core/tests/fixtures/utils/account/create.ts

```diff
@@ -55,6 +55,7 @@ export async function createAccount() {
               postal_code,
             },
           ],
+          origin_channel_id: process.env.BIGCOMMERCE_CHANNEL_ID,
```

Comment:
This is to ensure we create the account on the same channel we are making the register customer mutation to. In the future we need to be aware of multi-channel account creation issues.